docs/source/interpolation.rst

```diff
@@ -180,14 +180,17 @@ argument:
    :dedent:
    :lines: 72-73, 80
 
-By default the missing degrees of freedom (DoFs, the global basis function
-coefficients which could not be set) are zero:
+In this case, the default the missing degrees of freedom (DoFs, the global
```

Comment:
In this case, by default

Reply:
I've reworded, hopefully it's clearer now?